components/search/MoreFacetsDropdownOptions.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <b-dropdown-form>
+    <strong
+      class="mb-4 d-inline-block"
+    >
+      {{ $t(`facets.${facet.name}`).name }}
+    </strong>
+    <div class="option-group">
+      <MoreFacetsDropdownOption
+        v-for="(filter, indx) in [].concat(facet.fields).splice(0, limitTo)"
```

Comment:
"index"

Reply:
Lint complains that index is already defined in the upperscope